src/functions-reference/matrix_operations.Rmd

```diff
@@ -2359,6 +2392,18 @@ The left-singular vectors of `A`
 The right-singular vectors of `A`
 `r since("2.26")`
 
+<!-- tuple(matrix, vector, matrix); svd; (matrix A); -->
+\index{{\tt \bfseries svd }!{\tt (matrix A): tuple(matrix, vector, matrix)}|hyperpage}
+
+`tuple(matrix, vector, matrix)` **`svd`**`(matrix A)`<br>\newline
```

Comment:
I'm on the fence as to whether all these variants should say what the function does or just point out to other defs.